index.html

```diff
@@ -1,12 +1,732 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Document</title>
+  <title>SCSS-BEM</title>
+  <link rel="shortcut icon" type="image/png" href="./images/favicon.png"/>
+  <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.10.2/css/all.min.css" rel="stylesheet">
+  <link rel="stylesheet" href="css/slick.css">
+  <link rel="stylesheet" href="css/main.css">
 </head>
+
 <body>
-  <p>scss</p>
+  <!-- header -->
+  <header class="header">
+    <div class="menu">
+      <nav class="menu__left">
+        <ul class="nav">
+          <li class="nav__item">
+            <a href="#" class="nav__link">home <i class="fas fa-sort-down"></i></a>
```

Comment:
đoạn này đang để sai. Tất cả những item menu có icon sort-down tức là item menu đó có submenu. Như vậy thì việc thêm icon vào HTML là sai hoàn toàn. Nếu như hiện tại 1 item menu đang không có submenu, customer muốn thêm submenu cho item đó thì thêm icon sort-down vào HTML như thế nào?
Sửa toàn bộ lại phần bên dưới.